Formula/s/streamrip.rb

```diff
@@ -0,0 +1,217 @@
+class Streamrip < Formula
+  include Language::Python::Virtualenv
+
+  desc "Scriptable music downloader for Qobuz, Tidal, SoundCloud, and Deezer"
+  homepage "https://github.com/nathom/streamrip"
+  url "https://github.com/nathom/streamrip/archive/refs/tags/v2.0.5.tar.gz"
+  sha256 "83ee51192faeb8134ba29e772fee949de9778f6f9c7987abe421044e44b49bb4"
+  license "GPL-3.0-only"
+
+  depends_on "certifi"
+  depends_on "ffmpeg"
+  depends_on "pillow"
+  depends_on "python@3.13"
+
```

Comment:
I missed the linux dependency, 
```suggestion
  depends_on "python@3.13"

  uses_from_macos "libffi"

```